### PR TITLE
Edge type property index

### PR DESCRIPTION
**Pull request by Darych**
### Description

Please briefly explain the changes you made.

### Pull request type

Please check what kind of PR this is:

- [x] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to: 
https://github.com/memgraph/memgraph/pull/1964

Closes:
https://github.com/memgraph/memgraph/issues/662

### Checklist:

- [x] Check all content with Grammarly
- [x] Perform a self-review of my code
- [x] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [x] The build passes locally
- [ ] My changes generate no new warnings or errors
- [x] Add a corresponding label
- [x] If release-related, add a product and version label
- [x] If release-related, add release note on product PR

### CODE REVIEW

File: pages/fundamentals/indexes.mdx [hunk 1]
```diff
@@ -151,7 +151,41 @@ Creating an edge-type index will optimize the following type of queries:
 MATCH ()-[r:EDGE_TYPE]->() RETURN r;
 ```
 
-#### Analyze graph
+<Callout type="info">
+
+If you need to access nodes of found edges, you can use the `startNode(r)` and `endNode(r)` functions.
+Currently, named parameters are not supported for edge-type indexes.
```
Comment (hal-eisen-MG) [inline pages/fundamentals/indexes.mdx:157]: Remove "Currently"

File: pages/fundamentals/indexes.mdx [hunk 2]
```diff
@@ -151,7 +151,41 @@ Creating an edge-type index will optimize the following type of queries:
...
+Creating an edge-type property index will optimize the following type of queries:
+
+```cypher
+MATCH ()-[r:EDGE_TYPE {property_name: value}]->() RETURN r;
+```
+
+<Callout type="info">
+
+If you need to access nodes of found edges, you can use the `startNode(r)` and `endNode(r)` functions.
+Currently, named parameters are not supported for edge-type property indexes.
```
Comment (hal-eisen-MG) [inline pages/fundamentals/indexes.mdx:184]: Remove "Currently"

**Review by hal-eisen-MG — APPROVED** [review]
LGTM

**Comment by hal-eisen-MG** [comment]
> @hal-eisen-MG @kgolubic Which is better? With empty line in Info section image Or without? image

With the empty line please

**Review by hal-eisen-MG — APPROVED** [review]
LGTM

**Review by kgolubic — APPROVED** [review]
Everything looks OK to me.

**Pull request merged**